example/lib/company.ex

```diff
@@ -15,10 +15,10 @@ defmodule Company do
 
     has_many :people, Person
 
-    timestamps
+    timestamps()
   end
 
-  @optional_fields ~w(name is_active website city address facebook twitter founded_in)
+  @optional_fields ~w(name is_active website city address facebook twitter founded_in)a
```

Comment:
what does this a do here?

Reply:
Ecto 3.0 Ecto.Changeset.cast/3 with list of strings is deprecated, you have to pass list of atoms  
[https://github.com/elixir-ecto/ecto/blob/master/CHANGELOG.md#deprecations](https://github.com/elixir-ecto/ecto/blob/master/CHANGELOG.md#deprecations)